content/html/concepts/comments/comments.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Comments'
+Description: 'HTML comments are markup included in the HTML source that do not affect the rendering of the HTML document.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+HTML [comments](https://www.codecademy.com/resources/docs/general/comment) are markup included in the HTML source that do not affect the rendering of the HTML document. They are used for documenting the HTML code. They are also used for commenting out HTML that the developer doesn't want to render but does not desire to delete, either for historical or debugging purposes.
+
+## Syntax
+
+```html
+<!-- Comment goes here -->
+```
+
+An HTML comment can contain any other HTML markup between the opening `<!--` and the closing tag `-->` except for another comment. Attempting to nest HTML comments will result in unexpected behavior because the comment is closed at the first `-->` it encounters. All text after the first `-->` will render, including the outer enclosing `-->`. Thus, when commenting out HTML code, the developer must make sure any interior comments are removed or strip the trailing `-->`s.
```

Comment:
```suggestion
HTML comments always start with an opening `<!--` and end with a closing tag `-->`. They may contain anything from text to other markup that will not be rendered. 

Nested comments are not recommended because they may cause unexpected behavior. Comments generally end after the first closing tag `-->`. Anything after that will be rendered on the page, including all outer closing tags. Thus, when commenting out HTML code, developers should remove any interior comment tags (or at least the trailing ones `-->`).
```